--- gstreamermm-0.10.9/tools/Makefile.am.old	2011-04-15 10:30:41.000000000 +0200
+++ gstreamermm-0.10.9/tools/Makefile.am	2011-04-15 10:31:34.000000000 +0200
@@ -48,7 +48,7 @@ extra_defs_gen_generate_plugin_gmmproc_f
 
 extra_defs_gen_generate_defs_gst_SOURCES = extra_defs_gen/generate_defs_gst.cc
 extra_defs_gen_generate_defs_gst_LDADD = \
-	$(GSTREAMERMM_LIBS) -lglibmm_generate_extra_defs-2.4 $(local_lib)
+	$(GSTREAMERMM_LIBS) -lglibmm_generate_extra_defs-2.4 $(local_lib) -lglibmm-2.4 -lgstreamer-0.10
 
 AM_CPPFLAGS = -I$(top_builddir) $(GSTREAMERMM_CFLAGS)
 AM_CXXFLAGS = $(GSTREAMERMM_WXXFLAGS)
